can_msgs/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(can_msgs)
 
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
```

Comment:
Up to @ipa-mdl, but would you perhaps want to add `set(CMAKE_CXX_STANDARD_REQUIRED ON)` here as well?

Reply:
@gavanderhoorn: Thank for pointing this out.